src/models/passport/passport.ts

```diff
@@ -1,32 +1,100 @@
 import { get as getByKeyPath, has } from "lodash";
 
+// import { ServiceFiles } from "../../export/digitalPlanning/schema/types";
```

Comment:
These are intentionally commented out to allow this PR to proceed without a schema update - they'll be commented back in shortly.